Lib/test/test_platform.py

```diff
@@ -169,10 +169,12 @@ def test_uname_processor(self):
         of 'uname -p'. See Issue 35967 for rationale.
         """
         with contextlib.suppress(subprocess.CalledProcessError):
-            self.assertEqual(
-                platform.uname().processor,
-                subprocess.check_output(['uname', '-p'], text=True).strip(),
-            )
+            expect = subprocess.check_output(['uname', '-p'], text=True).strip()
+
+        if expect == 'unknown':
+            return
+
+        self.assertEqual(expect, platform.uname().processor)
```

Comment:
Not a huge deal, but this test reads a little bit strange to me since the typical format is `assertSomething(actual, expected)`. It has zero functional difference of course, but I find that it helps future readers when it's at least consistent within the same test file. At a brief glance, it looks like the other nearby tests use the `(actual, expected)` format.

```suggestion
        self.assertEqual(platform.uname().processor, expect)
```